centos-upgrade-second-stage.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+#
+# Script to run the second stage (everything after rebooting into the
+# newest kernel) for upgrading the CenOS AMIs from the imported
+# version to the latest in that major release series.
+#
+
+if test "`sed -n '/CentOS release 6\..*/p' /etc/centos-release`" != ""; then
+    is_centos6=1
+else
+    is_centos6=0
+fi
+
+if test $is_centos6 -eq 1; then
+    # upgrade to latest ENA
+    echo "Upgrading ENA module"
+
+    echo "Getting ENA source"
+    git clone https://github.com/amzn/amzn-drivers.git
+    cd amzn-drivers
+    git checkout ena_linux_1.3.0
+
+    echo "Building ENA source"
+    cd kernel/linux/ena
+    make
+
+    echo "Installing ENA source"
+    sudo cp ena.ko /lib/modules/`uname -r`/weak-updates/.
```

Comment:
Looking at the ena driver Makefile, it doesn't seem to support a make install. A better way to install it here is:

`make -C /lib/modules/$(BUILD_KERNEL)/build M=$(CURDIR) modules_install`

Reply:
Replace `$BUILD_KERNEL` with `uname -r` like you did earlier. I just took the line from the Makefile and added _install.

Reply:
Yeah, This is what we've done in the past, would rather not change that today....

Reply:
I don't think it will be intrusive to break anything. The modules_install will make sure the exported symbols get updated in the symbol table correctly for dependencies to use.

Reply:
I welcome a patch after we release CfnCluster 1.4.0.  We need to get this thing shipped today, and changes that are good but not necessary can wait.